src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+    <Configuration Condition=" '$(Configuration)' == '' ">$(OS)_Debug</Configuration>
```

Comment:
Lets try this for now but I suspect this isn't going to give us the full experience we want in all scenario.